reckoner/course.py

```diff
@@ -115,6 +114,19 @@ def plot(self, charts_to_install):
         for chart in self.charts:
             if chart.release_name in charts_to_install:
                 self._charts_to_install.append(chart)
+                charts_to_install.remove(chart.release_name)
+            else:
+                logging.debug(
+                    'Skipping {} in course.yml, not found '
+                    'in your requested charts list'.format(chart.release_name)
+                )
+
+        if len(self._charts_to_install) == 0:
+            raise NoChartsToInstall(
+                'No charts found from requested list ({}). They do not exist '
+                'in the course.yml. Verify you are using the release-name '
+                'and not the chart name.'.format(', '.join(charts_to_install))
+            )
```

Comment:
This could be a little confusing and might be better suited as a set of method. Basic concept is that for each chart we've asked to install, we'll add it to `self._charts_to_install` and remove it from `charts_to_install` (yes confusing I know, didn't want to refactor all the existing names of variables, really this should be called `requested_release_names`). 

Once we find that there is nothing in `self._charts_to_install` then we explode an error to the user.